convert2rhel/unit_tests/utils_test.py

```diff
@@ -347,3 +348,79 @@ def test_prompt_user(question, is_password, response, monkeypatch):
         monkeypatch.setattr(moves, "input", lambda _: response)
 
     assert prompt_user(question, is_password) == response
+
+
+@pytest.mark.parametrize(
+    ("content", "output"),
+    (
+        (
+            "[subscription_manager]\npassword = correct_password",
+            {"password": "correct_password", "activation_key": None},
+        ),
+        (
+            "[subscription_manager]\nactivation_key = correct_key\nPassword = correct_password",
+            {"password": "correct_password", "activation_key": "correct_key"},
+        ),
+        (
+            "[subscription_manager]\nincorrect_option = incorrect_content",
+            {"password": None, "activation_key": None},
+        ),
+        ("[INVALID_HEADER]\nactivation_key = correct_key", {"password": None, "activation_key": None}),
+        ("nonexisting", {"password": None, "activation_key": None}),
+    ),
+)
+def test_options_from_config_files_default(monkeypatch, content, output, tmp_path, caplog):
+    """Test config files in default path."""
+    path = os.path.join(str(tmp_path), "convert2rhel.ini")
+    tool_opts = ToolOpts()
+    if content != "nonexisting":
+        with open(path, "w") as file:  # pylint: disable=unspecified-encoding
+            file.write(content)
+        os.chmod(path, 0o600)
+
+    paths = ["/nonexisting/path", path]
+    monkeypatch.setattr(utils, "CONFIG_PATHS", value=paths)
+    tool_opts = utils.options_from_config_files(tool_opts)
+
+    assert tool_opts.password == output["password"]
+    assert tool_opts.activation_key == output["activation_key"]
+    if "INVALID_HEADER" in content:
+        assert "Unsupported header" in caplog.text
+    if "incorrect_option" in content:
+        assert "Unsupported option" in caplog.text
+
+
+@pytest.mark.parametrize(
+    ("content", "output"),
+    (
+        (
+            "[subscription_manager]\nactivation_key = correct_key\nPassword = correct_password",
+            {"password": "correct_password", "activation_key": "correct_key"},
+        ),
+    ),
+)
+def test_options_from_config_files_specified(monkeypatch, content, output, tmp_path, caplog):
```

Comment:
Minor: I would organize this so that the fixtures (moneypatch, tmp_path, and caplog) are grouped separately from the parameters handled by parametrize (content and output).